scripts/build.sh

```diff
@@ -73,3 +73,5 @@ distributions="$(dirname "${zipPath}")"
 
 echo "COPY ${distributions}/*.zip"
 cp ${distributions}/*.zip ./$OUTPUT/plugins
+
+./gradlew publishPluginZipPublicationToZipStagingRepository -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER
```

Comment:
Hey once you call this task `publishPluginZipPublicationToZipStagingRepository` make sure you run `cp -r ./build/local-staging-repo/org/opensearch/. $OUTPUT/maven/org/opensearch` 
Example : https://github.com/opensearch-project/job-scheduler/pull/182/files